app/models/page.rb

```diff
@@ -45,7 +45,7 @@ def submit
 
   def load_from_session(session, keys)
     keys.each do |key|
-      self.load("#{key}": session.dig(:page, key.to_sym) || send(key.to_sym))
+      self.load("#{key}": session.with_indifferent_access.dig(:page, key.to_sym) || send(key.to_sym))
```

Comment:
When testing locally I get the following error:

```
undefined method `with_indifferent_access' for #<ActionDispatch::Request::Session:0x000000011d384fc8 ...
```

Reply:
I'll fix this issue in another commit. lets get the original fix deployed

Reply:
https://github.com/alphagov/forms-admin/pull/645